english/pikachu/pokeball.md

```diff
@@ -0,0 +1,24 @@
+Suddenly one marshmallow turns into pikachu!
+
+And another turns into a pokeball!
+
+You decide to try to catch pikachu by throwing ball at it.
+
+Pikachu is trapped in ball but only lasts 2 seconds in it and pops out!!
+
+Pikachu is now angry and throws ball at you and apparently you are now trapped inside!!
+
+Its so dark and hot in the ball and you realize this is the end!
+
+But wait! this has to be a dream right?
+
+I cant be in a pokeball.
+
+They dont exist!!
+
+Its time to get up!
+
+I must resolve this marshmallow dream.
+I must get revenge from pikachu as well but maybe later....
+
+[Wake Up](awake/awake.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.